ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,18 @@
 Please **do not post any internal, closed source snippets** on this public issue tracker!
```

Comment:
I think it might be cleaner to move the *Issue_Template* and *PullRequest_Template* into a *.github/ISSUE_TEMPLATE* folder? 
See here for an example (https://github.com/hashicorp/terraform/tree/ec90bed62d008b5c1d9ce291a03226633a7e3069/.github)

I find large root directories so intimidating!

Reply:
We can also hide the top comment using `<!--` closed by `-->` tags.

Reply:
> I think it might be cleaner to move the _Issue_Template_ and _PullRequest_Template_ into a _.github/ISSUE_TEMPLATE_ folder?
> See here for an example (https://github.com/hashicorp/terraform/tree/ec90bed62d008b5c1d9ce291a03226633a7e3069/.github)
> 
> I find large root directories so intimidating!

I think so. I'd prefer do it in another PR. This one is for sysl binary info:)

Reply:
#556